### ISSUE TITLE
How do I get a completely minified version of this package?

### ISSUE THREAD

**Issue by tayyabmh** [walletlink/walletlink#157]
I am using jsdlvr and unpkg as the platform I'm building on doesn't allow me to perform an `npm install` therefore I am having to drop in packages as loaded scripts into the HTML head. 

- Loading static versions, not dynamic ones. At least trying.

Unfortunately, when I try to drop in "index.min.js" (https://cdn.jsdelivr.net/npm/walletlink@2.1.8/dist/index.min.js) I don't have WalletLinkProvider or exports as a package.

How do I get a version of this library that is completely contained and minified for production use that I statically load into an html head?

Thanks,

**Comment by junaid-faryad** [comment]
Hi @tayyabmh were you able to find a solution for the above issue?

**Comment by tayyabmh** [comment]
Yep! Need help or just asking?

**Comment by junaid-faryad** [comment]
@tayyabmh yeah, I'm also trying to integrate walletlink in a static html site. I'm facing issues with exports. I was wondering, were you able to make it work? I'd really appreciate, if you could please share any info regarding that.
Thank you

**Comment by tayyabmh** [comment]
Hey Junaid, what worked for me was to start a blank project in a separate directory, do an npm install of the walletlink node module, and then use browserify to get an outputted bundle file. I included that static bundle file into my static html project and it ended up working for me.

Let me know if that works for you, or if you need more help.

https://browserify.org

**Comment by ryan22271** [comment]
Listen to my words !some hack signed me up on this ! I’m not a programmer and I’m tired of being hacked by you all ! I have a computer but I have never done any of the stuff you all ask me to do and I won’t do it 


Sent from ME


On Tuesday, September 28, 2021, 9:44 AM, tayyabmh ***@***.***> wrote:




Hey Junaid, what worked for me was to start a blank project in a separate directory, do an npm install of the walletlink node module, and then use browserify to get an outputted bundle file. I included that static bundle file into my static html project and it ended up working for me.

Let me know if that works for you, or if you need more help.

https://browserify.org

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by junaid-faryad** [comment]
Thank you very much for the help!

**Comment by stevo2105** [comment]
https://whop.com/walletlink.js 

Spent a while on this
